src/main/java/uk/gov/hmcts/reform/civil/enums/settlediscontinue/SettleDiscontinueYesOrNoList.java

```diff
@@ -0,0 +1,13 @@
+package uk.gov.hmcts.reform.civil.enums.settlediscontinue;
```

Comment:
Why didn't we use existing YesOrNO datatype?

Reply:
We had to create a YesNo json in CCD to display the Yes/No vertical way. So, we have created similar object in Service as YesOrNo type is causing issue in building CaseData and midEvent is not triggered. We used a similar approach as done in existing fields. For eg: ExpertReportsSent